src/sage/schemes/hyperelliptic_curves/mestre.py

```diff
@@ -195,14 +195,14 @@ def HyperellipticCurve_from_invariants(i, reduced=True, precision=None,
         c233*F2*F3**2 + c333*F3**3
 
     try:
-        f = f*f.denominator()  # clear the denominator
+        f = f * f.denominator()  # clear the denominator
     except (AttributeError, TypeError):
         pass
 
     if reduced:
-        raise NotImplementedError("Reduction of hyperelliptic curves not " \
-                                   "yet implemented. " \
-                                   "See trac #14755 and #14756.")
+        raise NotImplementedError("Reduction of hyperelliptic curves not "
+                                  "yet implemented. "
+                                  "See github #14755 and #14756.")
```

Comment:
`issue` instead of `github`. See https://github.com/sagemath/sage/pull/35015#issuecomment-1422525031